### PR TITLE
Fixes pixi.js not autoupdating. #9699

### PR DESCRIPTION
**Pull request by Yennick**
Pull request for issue: #9699

This should fix the pixi.js npm autoupdate.

@PeterDaveHello

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@Yennick Thanks.

File: ajax/libs/pixi.js/package.json [hunk 1]
```diff
@@ -15,10 +15,11 @@
   "npmName": "pixi.js",
   "npmFileMap": [
     {
-      "basePath": "bin",
+      "basePath": "dist",
       "files": [
         "pixi.js",
-        "pixi.dev.js"
+        "pixi.min.js",
+        "pixi.min.js.map"
```
Comment (PeterDaveHello) [inline ajax/libs/pixi.js/package.json]: We could just use single `"**/*"` here to grab all the files, would you please use this rule to replace all the others? Thanks.

**Comment by PeterDaveHello** [comment]
@Yennick I'm very sorry, since I didn't notice your update and #9687 just been sent a little bit earlier, so I just merged #9687 first, still very thanks for your report and help, hope we can work on next PR together, thanks again!

**Comment by Yennick** [comment]
No problem, better luck next time ;)

**Comment by PeterDaveHello** [comment]
Thanks for your kindness @Yennick :)

**Pull request closed without merging**